menu/menu.yaml

```diff
@@ -182,6 +182,7 @@
           Url: "nservicebus/transports"
           Articles:
             - "nservicebus/transports/queuecreation"
+        - "nservicebus/transports/transactions"
```

Comment:
no, I intended this page to be on the "root" of the transports menu, not inside "Extending"